WebServices/TodoREST/TodoAPI/TodoAPI/TodoAPI.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
Any reason not to use 5.0?

Reply:
Right now I think we want to stick LTS because that is what is downloaded and installed by default.....